osidb/models.py

```diff
@@ -1973,6 +1973,16 @@ def is_community(self) -> bool:
             name=self.ps_module, ps_product__business_unit="Community"
         ).exists()
 
+    @property
+    def is_compliance_priority(self) -> bool:
+        """
+        check and return whether this affect module and component combination is compliance priority
```

Comment:
I would recommend to mention this file without the url to it since it is only accessible internally.